posit_team_contact_info.md

```diff
@@ -28,12 +28,28 @@ Authorisation for the request is required from the person's line manager, and wi
 
 ## All other types of support request
 
-Public Health Scotland's Data Science team will be providing first-line support for Posit Team applications and will be triaging all queries and requests for support with Posit applications.
+Public Health Scotland's Data Science team will be providing first-line support for Posit Team applications and will be triaging all queries and requests for support with Posit applications. Before raising a support request, please consider the following:
 
-To raise an issue, report an incident, make a change request or ask a question about Posit Team applications, please complete the form at [https://forms.office.com/e/pqmmNMkhT6](https://forms.office.com/e/pqmmNMkhT6).
+- [ ] **Check Error Messages**: Analytical servers generate error messages when something goes wrong, this is expected behaviour in many cases and can be related to the codebase, connected infrastructure such as databases or publishing platforms, or the system itself. Error messages can be confusing and steps-removed from the code, but can help you understand the issue and suggest the steps to take to resolve the issue. Make sure to read the error message carefully and try to understand its meaning, even just pasting into Google will often return similar problems _and solutions_.
```

Comment:
I wondered here something about the other issue that I've experienced was when I'd forgotten to type in those blasted "yes", "agree", "whatever" when prompted - this has given me all sorts of weird errors, perhaps that reminder also needs to go into trouble shooting? One instance was when loading a project (those prompts appear quite regularly!)